.github/workflows/CI.yml

```diff
@@ -12,7 +12,6 @@ jobs:
       matrix:
         version:
           - '1.6'
-          - '1'
```

Comment:
I suggest to have back this (erased) line, so we can test also Julia 1.9 as well. (I may have been who removed it...)